<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>530</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>DDOK DDOK</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(179, 236, 255)</string> <!--학생용 배경 색깔-->
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>360</width><!--360-->
     <height>1060</height><!--530-->
    </size>
   </property>
   <widget class="QLabel" name="label_sn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>

    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>

    <property name="text">
     <string>학      번</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_en">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>207</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>

    <property name="text">
     <string>시험번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_en">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
     </font>
    </property>
    <property name="accessibleName">
     <string>lineEdit_en</string>
    </property>
    <property name="styleSheet">        <!--시험번호 입력 칸 스타일시트-->
     <string notr="true">
     background-color:rgb(252, 252, 252);
     border: 2px solid gray;            <!--테두리-->
     border-radius: 10px;               <!--테두리 라운드 -->
     </string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="accessibleName">
     <string>enterButton</string>
    </property>
    <property name="styleSheet">            <!--시험 입장 버튼 스타일 시트-->
     <string notr="true">
     background-color:rgb(26, 198, 255);    <!--시험 입장 버튼 색깔 코드 255, 230, 153-->
     border: 2px solid rgb(26, 198, 255);     <!--테두리-->
     border-radius: 10px;                   <!--모서리-->
     </string>
    </property>
     <property name="font">
     <font>
      <family>나눔스퀘어라운드 Bold</family>
      <weight>75</weight>
     color: rgb(0,255,0);
     </font>
    </property>
    <property name="text">
     <string>시험 입장</string>
    </property>   
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquare</family>
     </font>
    </property>
    <property name="accessibleName">
     <string>lineEdit_sn</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">    <!--학번 입력 스타일시트 -->
     <string notr="true">
     background-color:rgb(252, 252, 252);
     border: 2px solid gray;        <!--테두리-->
     border-radius: 10px;           <!--테두리 라운드 -->
     </string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>80</x><!--130/ 글자 위치 맞추기-->
      <y>60</y><!--60-->
      <width>200</width><!--101 이 너비 부분을 올리면서 아래 pointsize 올리기-->
      <height>41</height><!--41-->
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family> <!--폰트 바꾸기 기존:나눔스퀘어_ac ExtraBold-->
      <pointsize>18</pointsize>     <!--학생용 프로그램 ddok ddok 기존:10 -->
      <weight>150</weight> <!--기존 75-->
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DDOK DDOK</string> <!--학생용 프로그램 DDOK DDOK 이름 출력-->
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
